core/src/main/scala/akka/persistence/cassandra/journal/CassandraStatements.scala

```diff
@@ -229,7 +229,7 @@ trait CassandraStatements {
         sequence_nr = ?
     """
 
-  private[akka] def deleteMessages = {
+  private[akka] def deleteMessages =
```

Comment:
personally I like the `{` when the single statement is big chunk of code

Reply:
I will remove the `RedundentBraces` so we can leave them in as and when we like them